src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.csproj.user

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please use dot files (e.g. .editorconfig, .gitignore, etc) from Git Extensions. We should be committing files like this

Reply:
In general, I agree that `.user`-files should not be committed. However, I think this here is a legit exception to the rule:
Our [Extensibility package](https://www.nuget.org/packages/GitExtensions.Extensibility/) hooks a PreBuild event into VS to download the GE portable build. The default parameters (which portable version to download and where to save it locally) are defined [here](https://github.com/gitextensions/gitextensions.extensibility/blob/625b98946f388cc334937610f6d7e0001d0f0beb/src/GitExtensions.Extensibility/build/net461/GitExtensions.Extensibility.targets#L4-L6) and passed to a [PS script](https://github.com/gitextensions/gitextensions.extensibility/blob/master/src/GitExtensions.Extensibility/tools/Download-GitExtensions.ps1) every time a build is run either locally or in CI. You don't see any of this in your VS solution, because it's hidden deep inside a referrenced NuGet package.
IMO, this flow is neither very common nor easy to understand. But it fits our needs.

You can override the default values from the NuGet package, in case you want to use e.g. a specific CI build or a local build instead. For example, I used it to test with https://github.com/gitextensions/gitextensions/pull/7785.
In order to have easy access to the override-able parameters, I think they should be included in each plugin with some meaningful default values.
But at the same time I do not want accidental commits of these parameters, just because I made some tests on my system. This would mess up both CI and other users' local configurations. That's why we started to use `.user` -files. The same is already in place for the [PluginTemplate](https://github.com/gitextensions/gitextensions.plugintemplate/blob/master/src/GitExtensions.PluginTemplate/GitExtensions.PluginTemplate.csproj.user) and the [PluginManager](https://github.com/gitextensions/gitextensions.pluginmanager/blob/master/src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj.user).

Reply:
Makes sense, thank you for the explanation